### ISSUE TITLE
Parralax rendering in Armory

### ISSUE THREAD

**Issue by montedre** [armory3d/armory#658]
Would it be possible to get the parralax rendering as used by Decal Machine work in Armory?  I would be happy to provide the node set up and what ever else I can. 

Thanks Monte 

p.s. not sure how to set this as a feature request

*(drag & drop zipped .blend file here)*

**Comment by zauberparacelsus** [comment]
I think those labels are assigned by Lubos

**Comment by adriansnetlis** [comment]
While parallax mapping node isn't a thing, you can make one yourself. Just use parallax maths and make a node group that handles that ;)

**Comment by montedre** [comment]
I am pretty sure that is what the creator of Decal machine has done, however right now Armory can not deal with it and throws an error.

**Comment by 8BitsGotByte** [comment]
Would also love to see this feature in a future release.

Did some testing and my guess is that armory is not (yet) using the tangent-, geometry- and/or camera data-nodes that is needed for parallax to work.

Here is the example in cycles. Armory looks like the top which is albedo- and normalmap only.
![cycles_parallax](https://user-images.githubusercontent.com/40199415/42873713-01f51f0e-8a80-11e8-8b7d-9baaf421e74c.png)

[parallaxDisplacement.zip](https://github.com/armory3d/armory/files/2205256/parallaxDisplacement.zip)

**Comment by adriansnetlis** [comment]
Yes, those nodes should be working if we want some more control over shading in Armory.

However, a native parallax node would be good (could be optimized better than manually created node groups).

**Comment by luboslenco** [comment]
Tangent node should be fixed. 🔨
